### PR TITLE
Move the section on parser writing to CONTRIBUTING

### PR DESCRIPTION
**Pull request by julik**
Move the documentation on writing parsers into CONTRIBUTING as the README should be a no-frills way of _using_ the library first.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -123,6 +111,7 @@ MyParser = ->(io) {
   parsed_witdh, parsed_height = io.read(8).unpack('VV')
   # ...and return the FileInformation::Image object with the metadata.
   FormatParser::Image.new(
+    format: :imga,
```
Comment (bermannoah) [inline CONTRIBUTING.md]: typo - image

Comment (bermannoah) [reply CONTRIBUTING.md]: or not?

**Comment by julik** [comment]
No, not a typo - it's an imaginary file format ;-)

**Comment by bermannoah** [comment]
We should still add a parser for it ✨

**Pull request merged**